languages/ruby/exercises/concept/nil/.meta/design.md

```diff
@@ -0,0 +1,40 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of nil in Ruby.
+
+## Learning objectives
+
+- Know that `nil` is a special Ruby object used to represent an "empty" or "default" value
```

Comment:
Is it "special"?  It acts like any other value object, it is immutable, it evaluates in a "boolean" way, it is assignable.  I wonder if its use is "special" but it is not.

Reply:
```suggestion
- Know that `nil` is a Ruby object used to represent an "empty" or "default" value
```